core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -426,6 +494,13 @@ static String toString(
               .add("useHttp2Preface", useHttp2Preface)
               .add("useHttp1Pipelining", useHttp1Pipelining);
 
+        if (maxNumEventLoopsPerHttp1Endpoint > 0) {
+            helper.add("maxNumEventLoopsPerHttp1Endpoint", maxNumEventLoopsPerHttp1Endpoint);
+        } else if (maxNumEventLoopsPerEndpoint > 0) {
+            helper.add("maxNumEventLoopsPerEndpoint", maxNumEventLoopsPerEndpoint);
+        }
+        helper.add("maxNumEventLoopsFunction", maxNumEventLoopsFunction);
```

Comment:
Perhaps we should add this only when the function is not the default?

Reply:
Yes, it was nullable and I changed it to use the default function and I forget about this. 😄